pkg/rtc/config.go

```diff
@@ -176,6 +176,10 @@ func NewWebRTCConfig(conf *config.Config, externalIP string) (*WebRTCConfig, err
 		subscriberConfig.RTCPFeedback.Video = append(subscriberConfig.RTCPFeedback.Video, webrtc.RTCPFeedback{Type: webrtc.TypeRTCPFBGoogREMB})
 	}
 
+	if rtcConf.UseICELite {
```

Comment:
Guessing we want a default value of `true` as we are enabling ICE lite now for all protocol versions that support it.